### PR TITLE
[5.2] Update some more docblocks

### PR DESCRIPTION
**Pull request by scrubmx**

### CODE REVIEW

File: src/Illuminate/Routing/Route.php [hunk 1]
```diff
@@ -267,6 +267,7 @@ protected function controllerMiddleware()
     /**
      * Get the parameters that are listed in the route / controller signature.
      *
+     * @param string $subClass
```
Comment (GrahamCampbell) [inline src/Illuminate/Routing/Route.php]: string|null

Comment (GrahamCampbell) [inline src/Illuminate/Routing/Route.php]: also, please correct the spacing

File: src/Illuminate/View/View.php [hunk 2]
```diff
@@ -76,6 +76,9 @@ public function __construct(Factory $factory, EngineInterface $engine, $view, $p
      *
      * @param  callable|null  $callback
      * @return string
+     *
+     * @throws \Exception
+     * @throws \Throwable
```
Comment (GrahamCampbell) [inline src/Illuminate/View/View.php]: Just put one of these, not both please.

**Comment by scrubmx** [comment]
@GrahamCampbell Should I squash down to 1 commit?

File: src/Illuminate/View/View.php [hunk 3]
```diff
@@ -76,6 +76,8 @@ public function __construct(Factory $factory, EngineInterface $engine, $view, $p
      *
      * @param  callable|null  $callback
      * @return string
+     *
+     * @throws \Exception
```
Comment (GrahamCampbell) [inline src/Illuminate/View/View.php]: Throwable please

**Pull request merged**